ui/src/app/components/flex.tsx

```diff
@@ -31,3 +31,7 @@ export const FlexRowWrap = (props) => {
     {props.children}
   </FlexRow>;
 };
+
+export const FlexDivider = (props) => {
```

Comment:
What's the intended use for FlexDivider? Is it just an element that goes into a flex container to create space?

Reply:
it's a spacer to push items apart. adding a more descriptive name and a comment